apps/shell/views/index.hbs

```diff
@@ -24,12 +25,24 @@
         'ctrl-v-paste': true,
         'ctrl-c-copy': true
       };
+      start();
+
 
     // hterm default preferences are defined here. Switch to the tag that we're using.
     // https://chromium.googlesource.com/apps/libapps/+/master/hterm/js/hterm_preference_manager.js
 
-    oodShell = new OodShell(terminalContainer, webSocketUrl, terminalPrefs);
+    oodShell = new OodShell(terminalContainer, socket, terminalPrefs);
```

Comment:
Instead of the changes here and in apps/shell/public/javascripts/ood_shell.1.js, maybe it would be simpler to use https://github.com/joewalnes/reconnecting-websocket